beaufortC.py

```diff
@@ -0,0 +1,29 @@
+def convert(txt, key):
```

Comment:
Probably change the function name, it'll be confusing to what is being converted.
You can make it BeaufortConvert or soemthing of that sort.